MANIFEST.in

```diff
@@ -1,11 +1,13 @@
 include *.rst
 include *.txt
-include *.py
-include buildout.cfg
 include tox.ini
-include .travis.yml
 include .coveragerc
 
-recursive-include src *
+include buildout.cfg
+exclude buildout.cfg
```

Comment:
Make up your mind please :)

Reply:
Ah!  Okay, can you add a brief comment explaining this?  Something like

```
# using include + exclude here to exclude buildout.cfg when building an sdist
# without producing "does not exist" warnings when installing from an sdist.
```

Reply:
(Personally, I would just include buildout.cfg in the dists -- less hassle, no harm.)

Reply:
Okay, you are right this is way easier.